R/position.nodes.radial.R

```diff
@@ -39,6 +39,49 @@ position.nodes.node.radiusial <- function(v, tree, extra.len, spread = 1) {
 	tau <- -(pi / 2.5);
 	vi <- v[v$parent == -1, ];
 
+	calculate.angles <- function(v, tree, spread) {
```

Comment:
Is there a way to consolidate this with the one in `position.nodes.R`?  is the handling for `fixed.angle` inherently different from `spread`?

Also the function name in line 36 should be `radial` not `radiusial`, which I think is a typo

Reply:
Also the indentation in line 42-43 is funky

Reply:
> Is there a way to consolidate this with the one in `position.nodes.R`? is the handling for `fixed.angle` inherently different from `spread`?

Yep, that's exactly what I'd like to do next! The priority here was just to separate the angle calculations from everything else so that improvements like that would be possible. Now that these functions are separated, it'll be much easier and more reliable to tweak the implementation.
 
> Also the function name in line 36 should be `radial` not `radiusial`, which I think is a typo

Indeed - will fix.